### PR TITLE
Customers achievments

### PR DESCRIPTION
**Pull request by Belashab**
Finished 2 blocks Customers and achevements. Should work with ane screen width.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,7 +15,10 @@
         <main class="main_container">
 
             <section class= "achievements">
```
Comment (SeLub) [inline index.html:17]: I recommend to change **section** to **div** container.
Read why: https://github.com/Belashab/layout/issues/35

File: index.html [hunk 2]
```diff
@@ -15,7 +15,10 @@
...
 
             <section class= "achievements"> 
-                <!-- Изменения, например -->
+                        <img src="images/achievements1.png" class="image_scale"></img> 
+                        <img src="images/achievements2.png" class="image_scale"></img>
+                        <img src="images/achievements3.png" class="image_scale"></img>
+                        <img src="images/achievements4.png" class="image_scale"></img>
             </section>
 
             <h2 class="headlight font_bold_32">Our Services</h2>
```
Comment (SeLub) [inline index.html:24]: You missed suTitle "Early stage company? Eligible applicants get all of our products for just $49 a month."
I recommend to use code from our previous template:
      <h2>Our Services</h2>
      <span>Early stage company? Eligible applicants get all of our products for just $49 a month.</span>

**Review by SeLub — APPROVED** [review]
ok

**Comment by SeLub** [comment]
преред закрытием ПР читайте комментарии в тексте

**Review by ArtemMiniuk — APPROVED** [review]
Все хорошо

**Pull request merged**